### ISSUE TITLE
Use containers statelessly

### ISSUE THREAD

**Issue by mrsimonemms** [britishredcrosssociety/local-lockdown#35]
WRT the recent PR #25, setting the `enableBookmarking()` function to `state` will not work in the long term.

Basically, all Docker app should be considered stateless. That means that any transient data (in this case, the `*.rds` files for the bookmarks) may be lost at any point in time - they certainly won't be available for more than an couple of hours.

Anything you wish to save long-term must be in an external data source - that could be a blob storage account or a database. **This is particularly important to note when it comes to the CSV data in this repo if you are apply any manipulations to it.**

**tl;dr** - any data written to disk **WILL** be lost.

I know that means that URLs will be ugly, but that's an unfortunate side effect. If you require a short URL, use a shortening service such as [bit.ly](https://bit.ly) before sending to anyone.

I have created a PR #34 to fix this and add some documentation on the concept of statelessness which you should all get familiar with. This issue is raised to highlight it further and act as an aide-memoire
